web_programming/fetch_github_info.py

```diff
@@ -1,26 +1,31 @@
 #!/usr/bin/env python3
-
 """
 Created by sarathkaul on 14/11/19
 
 Basic authentication using an API password is deprecated and will soon no longer work.
 Visit https://developer.github.com/changes/2020-02-14-deprecating-password-auth
 for more information around suggested workarounds and removal dates.
-"""
 
 
+Updated by lawric1 on 24/11/20
+
+Authentication will be made via access token.
+To generate your personal access token visit https://github.com/settings/tokens.
+"""
+
 import requests
 
 _GITHUB_API = "https://api.github.com/user"
 
 
-def fetch_github_info(auth_user: str, auth_pass: str) -> dict:
+def fetch_github_info(auth_token: str) -> dict:
```

Comment:
The return type hint needs to be more descriptive.

Use `from typing import Dict` and specify it as: `Dict[type(key), type(value)]`

As we probably don't know what GitHub is going to give us we should probably specify it as `Dict[Any, Any]`